pennylane_qiskit/qiskit_device.py

```diff
@@ -325,7 +345,7 @@ def _get_state(self, result, experiment=None):
 
         Args:
             result (qiskit.Result): result object
-            experiment (str): the name of the experiment to get the state for
+            experiment (str): Optional, the name of the experiment to get the state for. Default=None.
```

Comment:
```suggestion
            experiment (str or None): the name of the experiment to get the state for. Default=None.
```

Reply:
Also there is no need to include the default value here (unless there is complex behaviour that you want to discuss), as Sphinx will automatically extract it from the signature :)